crates/sui-types/src/messages.rs

```diff
@@ -1809,59 +1809,30 @@ pub enum ExecuteTransactionResponse {
 
 #[derive(Debug, Clone, Serialize, Deserialize)]
 pub struct EpochInfo {
-    /// The epoch number of this epoch info.
-    /// Although we could derive it from the epoch number of `next_epoch_committee`, a potential
-    /// byzantine node may set next_epoch_committee.epoch as 0, which will make it very
-    /// inconvenient to obtain the epoch number.
-    epoch: EpochId,
-    /// The committee of the NEXT epoch. The committee of the epoch identified by the `epoch`
-    /// field is not part of this data structure. Instead, it will always be in the previous epoch
-    /// data structure. The committee for the very first epoch would be the genesis committee, which
-    /// is not in any epoch data structure, but in the genesis blob.
-    /// It's important that we commit to the next epoch committee in the current epoch, so that we
-    /// know what committee to use when verifying the next epoch data structure.
-    next_epoch_committee: Committee,
-    /// The last checkpoint included in this epoch. The first checkpoint can always be derived
-    /// from the previous epoch. The first checkpoint of the first epoch would be 0.
-    last_checkpoint: CheckpointSequenceNumber,
+    /// The committee of this epoch.
```

Comment:
I would actually keen a sequential `EpochId` in the epoch info. As I understand the EpochInfo is the structure that is collectively signed by validators. As such this is also the structure light clients will need to download to have for sure the latest committee keys. These light client will want to know that whoever is providing the stream of certified EpochId does not skip one -- and therefore a sequence number is useful here (besides the next checkpoint number).

Reply:
There is a EpochId inside committee. Is that what you meant here?